static/css/_blockly-toolbox.scss

```
@@ -59,7 +59,7 @@
     border-width: thin;
     color: $brand-dark-gray;
     border-right: 0.063em solid;
-    width: 11em;	
+    width: 15em;	
```

In my opinion, it would be better to specify a `min-width` of `11em` rather than setting the `width` to `15em`. This will leave the toolbox width unchanged for most users while growing for users who's strings exceed the `11em` width. Double check this across browsers though.

```suggestion
    min-width: 11em;
```